### ISSUE TITLE
Lightbox on login modal should extend to footer

### ISSUE THREAD

**Issue by alexrcabral** [benetech/MathShare#1291]
Currently the modal on login still displays the footer and allows focus on elements in the footer. These should be grayed out along with the rest of the body of the the page.
